code/modules/mob/living/living_defines.dm

```diff
@@ -100,7 +100,12 @@
 	var/usable_hands = 2
 
 	var/list/pipes_shown = list()
-	var/last_played_vent
+	var/last_played_vent = 0
+	/// The last direction we moved in a vent. Used to make holding two directions feel nice
+	var/last_vent_dir = 0
+	/// Cell tracker datum we use to manage the pipes around us, for faster ventcrawling
```

Comment:
```suggestion
	/// Cell tracker datum we use to manage the pipes around us, for faster ventcrawling.
```

Reply:
Do I have to

Reply:
![image](https://user-images.githubusercontent.com/15794172/167100794-c78dbd60-cfef-46db-953c-6d7d0af2b8fa.png)

Reply:
I don't like using punctuation in my runon sentences alright?